MAINTAINERS.yml

```diff
@@ -742,6 +742,15 @@ Documentation:
     description: >-
         Inventek es-WiFi
 
+"Drivers: WiFi WINC1500":
```

Comment:
If we do not appoint a maintainer (and we assume maintainer-ship is covered in the parent area), this patch has no real meaning.
If the intention is that @Kludentwo becomes a collaborator here, they can simply be added directly in collaborators under "Drivers: WiFi":